release/models/bfd/openconfig-bfd.yang

```diff
@@ -682,6 +689,46 @@ module openconfig-bfd {
     }
   }
 
+  grouping bfd-configuration {
+    description
+       "Configuration parameters of BFD when it is enabled in protocols.";
+    leaf desired-minimum-tx-interval {
+      type uint32;
+      units microseconds;
+      description
+        "The minimum interval between transmission of BFD control
+        packets that the operator desires. This value is advertised to
+        the peer, however the actual interval used is specified by
+        taking the maximum of desired-minimum-tx-interval and the
+        value of the remote required-minimum-receive interval value.
+
+        This value is specified as an integer number of microseconds.";
+    }
```

Comment:
Are these leaves defined in RFC? Can you add a reference to the BFD rfc and section?

Reply:
Hi @s19nal . You are right, I can do the change you propose and reuse the grouping also from the original place.  My first intention was just that these leaves do make sense when BFD is configured at protocol level beyond current enable. And then, if these are ok, make the necessary optimizations. 

So, @dplore @s19nal , let me add the references and reuse the grouping